scripts/change_url

```diff
@@ -114,13 +114,21 @@ fi
 ynh_add_config --template="homeserver.yaml" --destination="/etc/matrix-$app/homeserver.yaml"
 ynh_add_config --template="log.yaml" --destination="/etc/matrix-$app/log.yaml"
 
+#=================================================
+# SECURE FILES AND DIRECTORIES
+#=================================================
+
+# Only setting permissions for the two config files updated above
+chown $synapse_user:root -R /etc/matrix-$app
+chmod u=rwX,g=rX,o= -R /etc/matrix-$app
+
 #=================================================
 # SETUP PERMISSIONS
 #=================================================
 
 ynh_script_progression --message="Configuring permissions..." --weight=1
 ynh_permission_url --permission=server_api --clear_urls
-ynh_permission_url --permission=server_api --url=$domain/_matrix --additional_urls=$server_name/.well-known/matrix \
+ynh_permission_url --permission=server_api --url=$domain/_matrix --add_url=$server_name/.well-known/matrix \
```

Comment:
Well seeing the upgrade script I think that this code is completly obsolte. At my point of view we should remove theses both lines:
```bash
ynh_permission_url --permission=server_api --clear_urls
ynh_permission_url --permission=server_api --url=$domain/_matrix --add_url=$server_name/.well-known/matrix \
```